### PR TITLE
Transposed VCF for recording ambiguous bases.

### PR DESCRIPTION
**Pull request by yceh**
Add auxiliary file to record ambiguous base.
If the auxiliary file conflict with protobuf, it will use the mutations inside the auxiliary file.

### CODE REVIEW

**Comment by yceh** [comment]
Had an Inconsistent allele at 27718 of Turkey/HSGM-6841/2021|EPI_ISL_2102061|2021-03-09, T in protobuf, A in VCF in 07/26 protobuf, still debugging.

**Comment by yceh** [comment]
Never mind, false positive.

File: CMakeLists.txt [hunk 1]
```diff
@@ -159,14 +159,22 @@ else()
...
+        src/matOptimize/transpose_vcf/transposed_vcf_patch.cpp
         ${patch_tree}
         ${New_Profitable_Moves_Enumerators}
         ${PROTO_SRCS}
         ${PROTO_HDRS}
         ${DETAILED_MUTATIONS_PROTO_SRCS}
         ${DETAILED_MUTATIONS_PROTO_HDRS}
         )
-        
+        add_executable(transpose_vcf
```
Comment (yatisht) [inline CMakeLists.txt:170]: We need to have `add_executable` in the if block as well (used for CenntOS).

**Comment by yceh** [comment]
Thanks! Didn't realize that the same sequence can appear in VCF in multiple days, I need to add some locking to handle it, and also a utility for deduplicating.

**Pull request merged**